.github/workflows/docker.yaml

```diff
@@ -0,0 +1,71 @@
+name: Docker Build HTML Image
+
+on:
+  # Allow manual trigger
+  workflow_dispatch:
+    inputs:
+      # Ask if docker depoyment needed when manually triggered
+      deploy:      
+        description: 'Deploy Docker Image (y/n)?'
+        required: true
+        default: 'n'
+  push:
+    tags:
+      - '[0-9].[0-9]+.[0-9]+' 
+      
+env:
+  DOCKER_REGISTRY_PATH: useblocks
+  IMAGE_NAME: sphinxneeds
+  BASE_IMAGE: sphinxdoc/sphinx:latest
+  NEEDS_VERSION: 0.7.8
```

Comment:
@danwos Is the current release version available during the pipeline run, to avoid hard-coding here? This is needed to version to docker image